packages/icons/config/fabric-icons-7.json

```diff
@@ -64,10 +64,6 @@
 			"name": "Starburst",
 			"unicode": "EF78"
 		},
-		{
-			"name": "ShareiOS",
```

Comment:
I'm not sure why ShareiOS is deleted. In the first round of external logo clean up we kept it, it's showing up on the icon subsetter tool, and I don't remember manually removing it. We might have to update this again here and in the cdn if we want it back in.

Reply:
We also found "EaseOfAccessLegacy" and "Diamond" icons missing. Doesn't have to be in this bump, but would be great if we could have them back in next bump.

Reply:
It would be great to not push this until we understand why they're missing. Otherwise we break partners.

Reply:
ShareiOS was a me bug, so got that fixed. Looks like screener registered some icons as removed that weren't really. The rest of the removed icons were logos that shouldn't have been in there in the first place. So once screener passes, I think we're good to go!